metagraph/tests/algorithms/test_embedding.py

```diff
@@ -18,16 +17,14 @@ def test_node2vec(default_plugin_resolver):
         nx_graph.add_edge(node, node + 1)
     nx_graph.add_edge(50, 80)  # have non-consecutive node ids
     graph = dpr.wrappers.Graph.NetworkXGraph(nx_graph)
-
     mv = MultiVerify(dpr)
-
     p = 1.0
     q = 0.5
     walks_per_node = 8
     walk_length = 8
     embedding_size = 25
-    epochs = 10_000
-    learning_rate = 1e-3
+    epochs = 2
+    learning_rate = 5e-2
     embedding = mv.compute(
```

Comment:
I had to change the parameters for this test again in https://github.com/metagraph-dev/metagraph/pull/135. I think the easiest thing to do here is to checkout the file from the main repo.